drafts/te-types-update/draft-ietf-teas-rfc8776-update.txt

```diff
@@ -5518,9 +5594,77 @@ Internet-Draft            TE Common YANG Types            September 2023
            "RFC 4115 Bandwidth Profile";
          reference
            "RFC4115: A Differentiated Service Two-Rate, Three-Color
+
+
+
+Busi, et al.               Expires 20 May 2024                [Page 100]
+
+Internet-Draft            TE Common YANG Types             November 2023
+
+
            Marker with Efficient Handling of in-Profile Traffic";
        }
 
+     // CHANGE NOTE: The identity path-metric-loss below has
+     // been added in this module revision
+     // RFC Editor: remove the note above and this note
+     identity path-metric-loss {
+       base path-metric-type;
+       description
+         "The path loss (as a packet percentage) metric type
+         encodes a function of the unidirectional loss metrics of all
+         links traversed by a P2P path.
+
+         The basic unit is 0.000003%,
+         where (2^24 - 2) or 50.331642% is the maximum value of the
+         path loss percentage that can be expressed.
+
+         Values that are larger than the maximum value SHOULD be
+         encoded as the maximum value.";
+       reference
+         "RFC8233: Extensions to the Path Computation Element
+         Communication Protocol (PCEP) to Compute Service-Aware Label
+         Switched Paths (LSPs);
+
+         RFC7471: OSPF Traffic Engineering (TE) Metric Extensions;
+
+         RFC8570: IS-IS Traffic Engineering (TE) Metric Extensions.";
+     }
+
+     // CHANGE NOTE: The identity path-metric-delay-variation below has
+     // been added in this module revision
+     // RFC Editor: remove the note above and this note
+     identity path-metric-delay-variation {
+       base path-metric-type;
+       description
+         "The path delay variation encodes the sum of the unidirectional
+         delay variation metrics of all links traversed by a P2P path.
+
```

Comment:
I'm not sure the above statement is correct. "The path delay variation is not sum of individual delay variation".

Reply:
We may want to use delay variation (DV) as a bound on per link DV.. So, computation can prune links that exceeds the DV bound?

Reply:
This is the definition I have found in [RFC8233](https://datatracker.ietf.org/doc/html/rfc8233#section-3.1.2)